dsgrid/cli/query.py

```diff
@@ -148,7 +147,6 @@ def create_project_query(
     dataset_id,
     filters,
     aggregation_function,
-    default_per_dataset_aggregation,
```

Comment:
Is this a feature no one is using?

Reply:
It was a feature that was removed from the code long ago.